src/ai/backend/manager/models/gql_models/image.py

```diff
@@ -449,23 +449,27 @@ async def batch_load_by_image_identifier(
 
     @overload
     @classmethod
-    def from_row(cls, row: ImageRow) -> ImageNode: ...
+    def from_row(cls, graph_ctx: GraphQueryContext, row: ImageRow) -> Self: ...
 
     @overload
     @classmethod
     def from_row(
-        cls, row: ImageRow, *, permissions: Optional[Iterable[ImagePermission]] = None
+        cls, graph_ctx, row: ImageRow, *, permissions: Optional[Iterable[ImagePermission]] = None
     ) -> ImageNode: ...
 
     @overload
     @classmethod
     def from_row(
-        cls, row: None, *, permissions: Optional[Iterable[ImagePermission]] = None
+        cls, graph_ctx, row: None, *, permissions: Optional[Iterable[ImagePermission]] = None
     ) -> None: ...
 
     @classmethod
     def from_row(
-        cls, row: ImageRow | None, *, permissions: Optional[Iterable[ImagePermission]] = None
+        cls,
+        graph_ctx,
```

Comment:
Where is graph_ctx used?

Reply:
It is used in `batch_multiresult_in_scalar_stream()`. This function allows asynchronous batch load